src/lib/stores/tick.ts

```diff
@@ -20,7 +20,9 @@ export default (() => {
   }
 
   function stop() {
-    clearInterval(get(interval));
+    const i = get(interval);
+    i && clearInterval(i);
```

Comment:
<img width="134" alt="Screenshot 2022-09-07 at 17 18 39" src="https://user-images.githubusercontent.com/158411/188915780-04c38917-b9f4-47a2-8dfb-192947197d62.png">

Reply:
> Let handle be a user-agent-defined integer that is *greater than zero* that will identify the interval to be set by this call

https://www.w3.org/TR/2011/WD-html5-20110525/timers.html#list-of-active-intervals:~:text=Let%20handle%20be%20a%20user%2Dagent%2Ddefined%20integer%20that%20is%20greater%20than%20zero%20that%20will%20identify%20the%20interval%20to%20be%20set%20by%20this%20call.

Gonna make it a proper if statement just to be sure anyway ;)